pkg/broker/rbac.go

```diff
@@ -101,6 +105,11 @@ func NewBrokerClusterRole() *rbacv1.Role {
 				APIGroups: []string{"discovery.k8s.io"},
 				Resources: []string{"endpointslices", "endpointslices/restricted"},
 			},
+			{
+				Verbs:     []string{"get", "list"},
```

Comment:
?

```suggestion
				Verbs:     []string{"get", "list", "watch"},
```

Reply:
I had that initially but the resulting RBAC fails, I’ll try again.

Reply:
```
[cluster1] Error creating SA for cluster: error binding sa to cluster role: rolebindings.rbac.authorization.k8s.io "cluster-cluster1-submariner-k8s-broker-cluster" is forbidden: user "system:serviceaccount:submariner-k8s-broker:submariner-k8s-broker-admin" (groups=["system:serviceaccounts" "system:serviceaccounts:submariner-k8s-broker" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
[cluster1] {APIGroups:[""], Resources:["secrets"], Verbs:["watch"]}
```

Reply:
I thought we needed it b/c we're watching `Secrets`?

Reply:
So did I, but it seems to work without it...